app/models/manageiq/providers/ansible_tower/configuration_manager/configuration_script/api_create.rb

```diff
@@ -0,0 +1,36 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationScript
+  module ApiCreate
+    def create_in_provider(manager_id, params)
+      manager = ExtManagementSystem.find(manager_id)
+      job_template = manager.with_provider_connection do |connection|
+        connection.api.job_templates.create(params)
+      end
+
+      # Get the record in our database
+      # TODO: This needs to be targeted refresh so it doesn't take too long
+      EmsRefresh.queue_refresh(manager, nil, true) if !manager.missing_credentials? && manager.authentication_status_ok?
```

Comment:
for now you'll need to call `EmsRefresh.refresh` which is a synchronous call; otherwise the next line you will not be able to find the record.